meet-data/napf/0903/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+NAPF,2010-03-07,USA,OH,Columbus,GNC Pro Performance Deadlift Championships
```

Comment:
The date was reported as `07.03.2009` in the results file.

Not sure about location validity.

Also the results file seems to indicate that this was an `IPF` sanctioned meet instead of `NAPF`.